src/components/Table/TableRow/TableRow.tsx

```diff
@@ -1,16 +1,46 @@
-import React, { FC } from "react";
-import { VibeComponentProps } from "../../../types";
+import React, { FC, useCallback, useRef } from "react";
 import styles from "./TableRow.module.scss";
+import { VibeComponentProps } from "../../../types";
 import { ITableCellProps } from "../TableCell/TableCell";
+import { useKeyEvent } from "../../../hooks";
+import { SELECTION_KEYS } from "../../../constants";
+import { NOOP } from "../../../utils/function-utils";
 
 export interface ITableRowProps extends VibeComponentProps {
+  highlight?: boolean;
+  onClick?: (event: React.MouseEvent | React.KeyboardEvent) => void;
   children?: React.ReactElement<ITableCellProps> | React.ReactElement<ITableCellProps>[];
   style?: React.CSSProperties;
 }
 
-const TableRow: FC<ITableRowProps> = ({ children, style = {} }) => {
+const TableRow: FC<ITableRowProps> = ({ highlight = false, onClick = NOOP, children, style = {} }) => {
+  const ref = useRef(null);
+  useKeyEvent({
```

Comment:
this is a bit tricky since our table rows aren't clickable. I didn't see it in the storybook nor in tests so I can't tell how it works

Reply:
It's our implementation for handling keyboard press on elements, it works the same as doing:
```
<div
  ...
  onKeyDown={someCallback}
```

without the overhead for checking specific keycodes

Reply:
but do we allow focusing on a row? because you probably need some "focus" indication before clicking

Reply:
yep, it's happening on
```
tabIndex={onClick ? 0 : -1}
```

Reply:
Looks like this
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/dbe335f6-6ab6-473a-86a7-c563c4afed59)
A bit cut from the sides